diff --git a/node_modules/slash-create/lib/command.js b/node_modules/slash-create/lib/command.js
index 42444d1..b8eaac0 100644
--- a/node_modules/slash-create/lib/command.js
+++ b/node_modules/slash-create/lib/command.js
@@ -158,8 +158,8 @@ class SlashCommand {
             throw new TypeError('Command name must be a string.');
         if (opts.name !== opts.name.toLowerCase())
             throw new Error('Command name must be lowercase.');
-        if (opts.name.length < 3 || opts.name.length > 32)
-            throw new RangeError('Command name must be between 3 and 32 characters.');
+        if (opts.name.length < 1 || opts.name.length > 32)
+            throw new RangeError('Command name must be between 1 and 32 characters.');
         if (typeof opts.description !== 'string')
             throw new TypeError('Command description must be a string.');
         if (opts.description.length < 1 || opts.description.length > 100)
diff --git a/node_modules/slash-create/src/command.ts b/node_modules/slash-create/src/command.ts
index 840babb..d940e98 100644
--- a/node_modules/slash-create/src/command.ts
+++ b/node_modules/slash-create/src/command.ts
@@ -229,8 +229,8 @@ class SlashCommand {
   static validateOptions(opts: SlashCommandOptions) {
     if (typeof opts.name !== 'string') throw new TypeError('Command name must be a string.');
     if (opts.name !== opts.name.toLowerCase()) throw new Error('Command name must be lowercase.');
-    if (opts.name.length < 3 || opts.name.length > 32)
-      throw new RangeError('Command name must be between 3 and 32 characters.');
+    if (opts.name.length < 1 || opts.name.length > 32)
+      throw new RangeError('Command name must be between 1 and 32 characters.');
 
     if (typeof opts.description !== 'string') throw new TypeError('Command description must be a string.');
     if (opts.description.length < 1 || opts.description.length > 100)
